src/core/server/saved_objects/service/lib/repository.test.js

```diff
@@ -1065,6 +1121,21 @@ describe('SavedObjectsRepository', () => {
 
       expect(onBeforeWrite).toHaveBeenCalledTimes(1);
     });
+
+    it(`accepts a custom refresh setting`, async () => {
+      callAdminCluster.mockReturnValue({ result: 'deleted' });
+      await savedObjectsRepository.delete('globaltype', 'logstash-*', {
+        refresh: false
+      });
+
+      expect(callAdminCluster).toHaveBeenCalledTimes(1);
+      expect(callAdminCluster).toHaveBeenCalledWith('delete', {
+        id: 'globaltype:logstash-*',
+        refresh: false,
```

Comment:
(here and in some other places) I think it would be better to use a `toMatchObject({refresh: true})` like you did in line 678